### PR TITLE
fix imageUrl not updated from /account REST call, remove UserService.java's extra whitespace

### PR DESCRIPTION
**Pull request by tezcane**
AccountResource.saveAccount()->UserService.updateUser(): imageUrl is now passed so it can be updated
UserService.java: "@Transactional(readOnly = true)    " to "@Transactional(readOnly = true)"

new issues

### CODE REVIEW

File: generators/server/templates/src/main/java/package/service/_UserService.java [hunk 1]
```diff
@@ -263,6 +269,9 @@ public void updateUser(String firstName, String lastName, String email, String l
                 <%_ if (databaseType == 'mongodb' || databaseType == 'cassandra') { _%>
                 userRepository.save(user);
                 <%_ } _%>
+                <%_ if (searchEngine == 'elasticsearch') { _%>
+                // userSearchRepository.save(user); // left out by design
```
Comment (deepu105) [inline generators/server/templates/src/main/java/package/service/_UserService.java]: I dont see why this is required here. can you please rebase this branch with latest master and then remove this?

Comment (tezcane) [inline generators/server/templates/src/main/java/package/service/_UserService.java]: Sure.  Can you please help me understand why it is not needed and/or recommended?

Only the admin is calling userUpdate(UserDTO user) currently, does that have something to do with it?

It had resolved "User managment" page not getting updates in my code, though I really have my doubts now (will revisit).

**Comment by deepu105** [comment]
Plz rebase and squash

**Comment by tezcane** [comment]
OK, working on it now.

**Pull request closed without merging**